test/absinthe/execution/arguments_test.exs

```diff
@@ -24,6 +24,16 @@ defmodule Absinthe.Execution.ArgumentsTest do
     )
   end
 
+  @graphql """
+  query {
+    entities(representations: [{__typename: "Product", id: "123"}])
+  }
+  """
+  test "does not give error for object scalars in query or response" do
+    assert {:ok, %{data: %{"entities" => [%{"__typename" => "Product", "id" => "123"}]}}} =
+             run(@graphql, @schema)
+  end
```

Comment:
Can you include a test cases that does an ENUM literal, as well as test cases that utilize variables?

Reply:
Oh and also a `null` literal.

Reply:
As nested values in the scalar?